portfolio/src/main/webapp/script.js

```diff
@@ -39,24 +39,18 @@ function autoChange (){
 
 setInterval(autoChange, 5000);
 
-function getName() {
-  fetch("/data").then(response => response.text()).then((name) => {
-    document.getElementById('name').innerText = name;
-  });
-}
-
-function makeList(){
-    fetch("/data").then(response => response.json()).then((wordList) => {
-        const buildList = document.getElementById("make-me");
-        buildList.innerHTML = "";
-        buildList.appendChild(createli(wordList[0]));
-        buildList.appendChild(createli(wordList[1]));
-        buildList.appendChild(createli(wordList[2]));
+function addComment() {
```

Comment:
This seems like you're displaying comments, not adding one, right?
Renaming to loadComments() or showComments() or something might be more intuitive.